Resources/doc/reference/configuration/headers.rst

```diff
@@ -218,12 +249,22 @@ This value must be a valid input to ``DateTime``.
                     headers:
                         last_modified: "-1 hour"
 
-.. hint::
+.. note::
 
     Setting an arbitrary last modified time allows clients to send
     ``If-Modified-Since`` requests. Varnish can handle these to serve data
     from the cache if it was not invalidated since the client requested it.
 
+    If you only use these simple configurations to add cache validation
+    information, there is no point in adding both a last modified and ETag
+    header. Usually, you should prefer the ETag method as it is not prone to
+    clock skew issues.
```

Comment:
indeed. thanks for looking those up! ok, i will relent from giving any opinion and instead just add the links to the rft here. 
i don't understand the why, however. they basically say in 13.3.4 that the client should never change the last modified value in any way, so it becomes semantically the same as an etag to me. maybe twitter will find some answers: https://twitter.com/dbu/status/596594325821198336